app/views/pages/accessibility.md

```diff
@@ -3,60 +3,90 @@ page_title: Accessibility statement for Register trainee teachers
+<p class='govuk-body'>This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.</p>
 
-<h3 class='govuk-heading-s'>Non-compliance with the accessibility regulations</h3>
+<p class='govuk-body'>We’re waiting on third parties to assist with remediation so a fix date cannot be provided.</p>
 
-<p class='govuk-body'>To disable auto-fill on text inputs, we need to set the autocomplete attribute to a non-standard value. This fails <a href='https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html' class='govuk-link'> WCAG 1.3.5: Identify Input Purpose</a>.</p>
+<h3 class='govuk-heading-s'>Unsupported ARIA attributes are present on some form inputs</h3> 
+<p class='govuk-body'>This may cause issues for users of assistive technology. This fails WCAG 2.2 success criterion 4.1.2 Name, Role, Value.</p>
 
-<p class='govuk-body'>Auto-fill is not suitable for Register trainee teachers and makes the task of registering trainee teachers more difficult.</p>
+<p class='govuk-body'>We’re waiting on third parties to assist with remediation so a fix date cannot be provided.</p>
 
-<p class='govuk-body'>We believe the impact of a non-standard value is minimal and the value of disabling auto-fill is much greater. The fail is unlikely to impact our users and will benefit many of them.</p>
+<h3 class='govuk-heading-s'>While navigating with JAWS, parts of error messages are announced multiple times</h3>
+<p class='govuk-body'>This fails WCAG 2.2 success criterion 4.1.2 Name. Role, Value.</p>
 
-<p class='govuk-body'><a href='https://bat-design-history.netlify.app/register-trainee-teachers/preventing-browser-auto-fill/' class='govuk-link'>Read more about why and how we’ve prevented auto-fill in Register</a></p>
+<p class='govuk-body'>We’re waiting on third parties to assist with remediation so a fix date cannot be provided.</p>
 
+<h2 class='govuk-heading-m'>Disproportionate burden</h2>
+
+<p class='govuk-body'>We have determined that there is a disproportionate burden to meet all the requirements for the following reasons:
+
+<h3 class='govuk-heading-s'>Interactive tools</h3>
+
+<p class='govuk-body'>The upload file function is unavailable whilst navigating using speech recognition software. This is a known flaw of speech recognition software.</p>
+
+<p class='govuk-body'>We’ve assessed the cost of fixing the navigation issues and accessing information, as well as the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class='govuk-link' href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" target="_blank">disproportionate burden (opens in a new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.</p>
```

Comment:
Add the `rel="noreferrer noopener"` as suggested by the [design system](https://design-system.service.gov.uk/styles/links/#external-links).

```suggestion
<p class='govuk-body'>We’ve assessed the cost of fixing the navigation issues and accessing information, as well as the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class='govuk-link' href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" rel="noreferrer noopener" target="_blank">disproportionate burden (opens in new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.</p>
```